entity-management-web/src/integration-test/java/eu/europeana/entitymanagement/solr/SolrServiceTest.java

```diff
@@ -0,0 +1,68 @@
+package eu.europeana.entitymanagement.solr;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import eu.europeana.entitymanagement.common.config.AppConfigConstants;
+import eu.europeana.entitymanagement.config.AppConfig;
+import eu.europeana.entitymanagement.definitions.model.Agent;
+import eu.europeana.entitymanagement.definitions.model.Entity;
+import eu.europeana.entitymanagement.serialization.JsonLdSerializer;
+import eu.europeana.entitymanagement.solr.model.SolrAgent;
+import eu.europeana.entitymanagement.solr.model.SolrEntity;
+import eu.europeana.entitymanagement.solr.service.SolrService;
+import eu.europeana.entitymanagement.solr.utils.SolrEntityUtils;
+import org.junit.jupiter.api.Assertions;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import javax.annotation.Resource;
+
+import static eu.europeana.entitymanagement.testutils.BaseMvcTestUtils.AGENT_JSON;
+import static eu.europeana.entitymanagement.testutils.BaseMvcTestUtils.loadFile;
+
+/**
+ * JUnit test for testing the EMControllerTest class
+ */
+//@ContextConfiguration(classes = { EntityManagementApp.class})
+@SpringBootTest
+public class SolrServiceTest {
+
+    @Resource(name=AppConfig.BEAN_EM_SOLR_SERVICE)
+	SolrService emSolrService;
+    
+    @Qualifier(AppConfigConstants.BEAN_JSON_MAPPER)
+    @Autowired
+    private ObjectMapper objectMapper;
+    
+    @Autowired
+    JsonLdSerializer emJsonldSerializer;
+
+    //@Test
+    public void storeEntitiesInSolr() throws Exception {
```

Comment:
please create one test for each entity class

Reply:
Done.